src/channels.c

```diff
@@ -410,21 +410,21 @@ const idclass_t channel_class = {
     {
       .type     = PT_INT,
       .id       = "dvr_pre_time",
-      .name     = N_("DVR Pre"), // TODO: better text?
+      .name     = N_("DVR start padding"), // TODO: better text?
       .off      = offsetof(channel_t, ch_dvr_extra_time_pre),
       .opts     = PO_ADVANCED
     },
     {
       .type     = PT_INT,
       .id       = "dvr_pst_time",
-      .name     = N_("DVR Post"), // TODO: better text?
+      .name     = N_("DVR end padding"), // TODO: better text?
       .off      = offsetof(channel_t, ch_dvr_extra_time_post),
       .opts     = PO_ADVANCED
     },
     {
       .type     = PT_INT,
       .id       = "epg_running",
-      .name     = N_("Use EPG Running State"),
+      .name     = N_("Use EPG running state (accurate recordings)"),
```

Comment:
I think it might be better to just leave it as "Use EPG running state" or change it completely to "Use accurate  recording"?